draft-ietf-jsonpath-iregexp.md

```diff
@@ -124,14 +124,32 @@ In particular, full Unicode support is REQUIRED; the implementation
 MUST NOT limit itself to 7- or 8-bit character sets such as ASCII and
 MUST support the Unicode character property set in character classes.
 
+## Checking I-Regexp Implementations {#checking}
+
+A *checking* I-Regexp implementation only accepts the I-Regexp syntax
```

Comment:
This is OK, but I think the real goal is to be informed of potential problems.  You might choose to go ahead with a nonconforming regexp anyhow. So another definition could be "A checking I-Regexp implementation is one that has the capability of checking a submitted regexp for compliance with this specification and reporting any problems."

Reply:
OK, this exposes the fact that even  checking implementation might not be configured to check, if we define it along its capabilities; I'd like to reserve the term for one that actually does.
(And I'm not so hot about "submitted".)

➔ how about:

A *checking* I-Regexp implementation is one that checks a supplied regexp for compliance with this specification and reports any problems.

Reply:
Works for me.